web/styles/modal.css

```diff
@@ -383,3 +383,81 @@
             0 0 8px hsl(206deg 80% 62% / 60%);
     }
 }
+
+#add-poll-modal {
+    /* this height allows 3-4 option rows
+    to fit in without need for scrolling */
+    height: 450px;
```

Comment:
Does this behave reasonably in not-very-tall windows?

Reply:
Yes; in window heights shorter than 450px too, the entire modal fits inside and the scrollable options section shrinks but remains scrollable